README.md

```diff
@@ -47,4 +47,6 @@ Currently, two official plugins are available:
 - Michael Kleinschmidt ([@miklesch](https://github.com/miklesch))
 - Kalina Iwaszko ( [@messkalina] (https://github.com/messkalina))
 - Bhagya Samarathunga ( [@BhagyaPrasadSamarathunga] (https://github.com/BhagyaPrasadSamarathunga))
+- Bhagya Samarathunga ( [@BhagyaPrasadSamarathunga] (https://github.com/BhagyaPrasadSamarathunga))
```

Comment:
This seems like an unintentional change. Please remove.

Reply:
I did it actually. 
I removed the duplicated lines and pushed the new code with my name.
So it should show my name instead of Bhagya again.

Reply:
![Screenshot 2024-09-23 at 20 25 56](https://github.com/user-attachments/assets/d99bd424-df4f-4cda-b922-736b71796f99)